Orange/classification/naive_bayes.py

```diff
@@ -57,9 +57,10 @@ def predict_storage(self, data):
                 np.log(self.class_prob) +
                 np.array([np.zeros_like(self.class_prob)
                           if isnan(ins.x).all() else
-                          np.sum(attr_prob[:, int(attr_val)]
-                                 for attr_val, attr_prob in zip(ins, self.log_cont_prob)
-                                 if not isnan(attr_val))
+                          np.sum([attr_prob[:, int(attr_val)]
+                                  for attr_val, attr_prob in
+                                  zip(ins, self.log_cont_prob)
+                                  if not isnan(attr_val)], axis=0)
```

Comment:
Just changing `np.sum` to `sum` would be shorter and I think clearer, than creating a python list, and then using axis (I have to think about axis=0/1 every time).
But this works just as well.

Reply:
I removed this commit: naive Bayesian classifier is modified in #3540 and this patch is moved there to prevent conflicts.

I used `sum`, as suggested.